docs/sources/write/style-guide/ux-writing/index.md

```diff
@@ -83,7 +83,7 @@ Use active voice to make clear who is performing the action.
 
 ### Use sentence case in UI elements
 
-Capitalize only the first word in the title, the first word in a subheading after a colon, and any proper nouns.
+Capitalize only the first word in the title, the first word in a subheading after a colon, and any proper nouns ([product names]({{< relref "../capitalization-punctuation#capitalizing-proper-nouns" >}}) are considered proper nouns).
```

Comment:
nit: One of our Vale rules is to use parentheses judiciously. Is this better as a separate sentence?

```suggestion
Capitalize only the first word in the title, the first word in a subheading after a colon, and any proper nouns.
[Product names]({{< relref "../capitalization-punctuation#capitalizing-proper-nouns" >}}) are considered proper nouns.
```

Also not sure if the link should be on "Product names", "proper nouns", or if we should have an additional sentence with the "..., refer to [Capitalizing proper nouns](LINK)".
Is there a rule of thumb or is it more creative than that?

Reply:
I initially had it as a separate sentence, but it felt too prominent too me. Like it was becoming the point of things (especially with the blue link text) rather than an extra thought.

Reply:
Happy to go with whatever you feel is best, it's definitely useful to have this clarification!

Reply:
I'll run it by folks in the style roundtable as well.